src/components/pages/LandingPage/LandingPage.js

```diff
@@ -28,12 +28,14 @@ const LandingPage = () => {
             disadvantaged background.
           </p>
           <div className="content-container">
-            <div className="button-container-signin">
-              <h3>Already have an account?</h3>
-              <Button type="primary" onClick={() => loginWithRedirect()}>
-                Log In
-              </Button>
-            </div>
+            {!isAuthenticated /** IF the user is logged in, Hide the button */ ? (
+              <div className="button-container-signin">
+                <h3>Already have an account?</h3>
+                <Button type="primary" onClick={() => loginWithRedirect()}>
+                  Log In
+                </Button>
+              </div>
+            ) : null}
 
             <h3>Ready to Join Underdog Devs?</h3>
```

Comment:
Let's go ahead and include the "Ready to Join" CTA and buttons to this conditional rendering. Once we're logged in, we don't need these anymore!